apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -0,0 +1,37 @@
+import { LinkLibrary } from "@thunderstore/cyberstorm";
+import React from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps {
+  queryParams?: string;
+  url: string;
+}
+
+const Link = (props: LinkProps): React.ReactElement => {
+  const { children, queryParams, url } = props;
+  const q = queryParams ? `?${queryParams}` : "";
+
+  return <a href={`${url}${q}`}>{children}</a>;
+};
+
+const library: LinkLibrary = {
```

Comment:
These are not valid links I'm assuming? If so, it could be better to use the `noop` default link implementation instead of pretending we have a working one.

I don't consider this a blocker, but do create a task for setting up the links for storybook if you choose to not address it now.

Reply:
I'll add a separate task and sort those out in the end. Which kinda means I'm going to remove unused ones.